src/secp256k1/contrib/lax_der_parsing.c

```diff
@@ -0,0 +1,150 @@
+/**********************************************************************
```

Comment:
This will cause the following failure in `git-subtree-check.sh` (which admittedly we don't run on any subtree at the moment):

```
$ ./contrib/devtools/git-subtree-check.sh src/secp256k1
src/secp256k1 in HEAD was last updated to upstream commit 84973d393ac240a90b2e1a6538c5368202bc2224 (tree 0cedbba829a5b38b750b2952af49b45880c32117)
src/secp256k1 in HEAD currently refers to tree df87d7712d4efbb16439ea0cb108993d7cdc1dd8
:000000 040000 0000000000000000000000000000000000000000 fdb52a23d1e0770c2637cf926eb07e24d0683910 A	contrib
FAIL: subtree directory tree doesn't match subtree commit tree
```